src/Sarif/Processors/Log/SarifLogPipeline.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.CodeAnalysis.Sarif.Processors
     [Serializable]
     public class SarifLogPipeline
     {
-        [JsonRequired]
+        //[JsonRequired]
```

Comment:
>JsonRequired [](http://example.com/codeflow?start=11&length=12)

Still looking at this. Minimally, we can permit it for netstandard2.0 and elide otherwise.